Userland/Libraries/LibJS/Bytecode/Op.h

```diff
@@ -992,6 +1071,21 @@ ALWAYS_INLINE void Instruction::replace_references(BasicBlock const& from, Basic
 #undef __BYTECODE_OP
 }
 
+ALWAYS_INLINE void Instruction::replace_references(Register from, Register to)
+{
+#define __BYTECODE_OP(op)       \
+    case Instruction::Type::op: \
+        return static_cast<Bytecode::Op::op&>(*this).replace_references_impl(from, to);
+
+    switch (type()) {
+        ENUMERATE_BYTECODE_OPS(__BYTECODE_OP)
+    default:
+        VERIFY_NOT_REACHED();
+    }
+
+#undef __BYTECODE_OP
+}
```

Comment:
Since most implementation are empty isn't it easier to only dispatch to ops which actually do something with it?

Reply:
Oh just noticed the other replace does it like this as well
In that case we can clean it up later

Reply:
Later™ seems like a good time

Reply:
Yes indeed!